LocalSettings.php

```diff
@@ -2162,6 +2163,11 @@
 				'read' => true,
 			),
 		),
+		'+socioatlaswiki' => array(
+			'sysop' => array(
+				'moderation' => true,
```

Comment:
moderation isn't a restricted right

Reply:
@JohnFLewis ok, so why does next to wgEnableModeration it says “don’t forget to assign ‘moderation’ rights!”

Reply:
@Amanda-Catherine because it was installed before ManageWikiPermissions was a thing, and the comment was never removed